common/types/block_test.go

```diff
@@ -57,6 +58,12 @@ func TestStringToNodeID(t *testing.T) {
 	_, err = StringToNodeID(string(pubkey[:10]))
 	r.Error(err, "Expected error converting too-short string to NodeID")
 
+	//Test too long : make sure that everything after the first 64 characters is ignored
+	longString := nodeID1.String() + string(pubkey2[:])
```

Comment:
Insert after this line:
`r.Greater(len(longString), len(nodeID1.String()))`

(it's fairly obvious but it makes the test more readable and helps protect against future changes to the test - it's always good to be extra verbose in tests)